tools/cli/workflow_commands.go

```diff
@@ -222,9 +222,9 @@ func showHistoryHelper(c *cli.Context, wid, rid string) {
 	if err != nil {
 		if _, ok := err.(*types.EntityNotExistsError); ok {
 			fmt.Printf("%s %s\n", colorRed("Error:"), err)
-			return
+			return nil
 		}
```

Comment:
hmm.  is this actually not an error path?

seems fine to leave if you're not confident, since that doesn't change the current behavior.  just odd looking.

Reply:
actually this seems to match `ErrorAndPrint("text", nil)` behavior.  might be worth moving to that for consistency.

Reply:
Fixed it's definitely an old bug. It should fail for show history cli.